src/Image.php

```diff
@@ -28,6 +28,10 @@ class Image implements ImageDriver
     public function __construct(protected ?string $pathToImage = null)
```

Comment:
Should this be a property on the `Image` object? It isn't actually used anywhere but it is protected so I guess removing is a BC break.

If we keep, `Image::loadFile()` should probably set this property as well?